gapis/service/service.proto

```diff
@@ -979,6 +979,8 @@ message Context {
   path.API API = 2;
   // The estimated importance for the context. 0 = lowest priority.
   uint32 priority = 3;
+  // The context ID. 0 or 1 for GLES. 1 for Vulkan
```

Comment:
This comment is not correct, a GLES application can have more than two contexts (e.g. google maps have contexts 0, 1, 2).